### DISCUSSION TITLE
关于 Heo 的 "最新发布" 的排列方法

### DISCUSSION THREAD

**Discussion by larevelad** [tangly1024/NotionNext#3037] [Q&A] [answered]
Heo 主题右侧的 "最新发布" 是按修改时间排列的, 如何能让其按文章的创建时间来排列呢 ?

**Comment by Daleveral** [comment, marked as answer]
相关代码是 `lib/db/getSiteData.js` 中 latestPosts 函数处, 有 :

```JavaScript
    const dateA = new Date(a?.lastEditedDate || a?.publishDate)
    const dateB = new Date(b?.lastEditedDate || b?.publishDate)
```

改为 :

```JavaScript
    const dateA = new Date(a?.publishDate)
    const dateB = new Date(b?.publishDate)
```

如此就能安装发布时间排列, 当然还要修改显示的时间 : 

在 `themes/heo/components/LatestPostsGroupMini.js` 文件的末尾有 :

`<div className='text-gray-400'>{post.lastEditedDay}</div>` 

将 `post.lastEditedDay` 改为 `post?.publishDay` 即可
